katacomb/scripts/continuum_image.py

```diff
@@ -160,7 +160,7 @@ def main():
     parser = create_parser()
     args = parser.parse_args()
     configure_logging(args)
-    log.info(f"Reading data with applycal={args.applycal}")
```

Comment:
Yes, this is one place where it is sad to see f-strings go :joy:

So I guess katacomb is now Python 3.6+?

Reply:
Yes. The real achievement is the Obit is 😏.